uwblueprint/childrens-aid-society
# Juthika/health setup gh actions

## Notion ticket link

[Setup GH Actions for testing](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* added a docker-compose.ci file that will be called on docker-compose with the env variables from the ci.yml file
* created a ci.yml file to build the docker compose stack & run the tests
* created a script to run tests



## Steps to test
1. Make a PR!
2. Go to Actions under the Github UI
3. check to see if docker-compose worked & tests passed!



## What should reviewers focus on?
* whether or not the tests pass when making a new PR


## Checklist
